tests/test_plugins.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2020 Satoru SATOH
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import unittest
+import warnings
+
+try:
+    from unittest import mock
+except ImportError:  # for Python 2.7
+    mock = False
+
+from tests.yamllint_plugin_example import rules as example
+
+import yamllint.plugins
+import yamllint.rules
+
+
+PLUGIN_RULES = dict(example.RULES)
+
+
+class FakeEntryPoint(object):
+    """Fake object to mimic pkg_resources.EntryPoint.
+    """
+    RULES = example.RULES
+
+    def load(self):
+        """Fake method to return self.
+        """
+        return self
+
+
+class BrokenEntryPoint(FakeEntryPoint):
+    """Fake object to mimic load failure of pkg_resources.EntryPoint.
+    """
+    def load(self):
+        raise ImportError("This entry point should fail always!")
+
+
+class PluginFunctionsTestCase(unittest.TestCase):
+    def test_validate_rule_module(self):
+        fun = yamllint.plugins.validate_rule_module
+        rule_mod = example.forbid_comments
+
+        self.assertFalse(fun(object()))
+        self.assertTrue(fun(rule_mod))
+
+    @unittest.skipIf(not mock, "unittest.mock is not available")
+    def test_validate_rule_module_using_mock(self):
+        fun = yamllint.plugins.validate_rule_module
+        rule_mod = example.forbid_comments
+
+        with mock.patch.object(rule_mod, "ID", False):
+            self.assertFalse(fun(rule_mod))
+
+        with mock.patch.object(rule_mod, "TYPE", False):
+            self.assertFalse(fun(rule_mod))
+
+        with mock.patch.object(rule_mod, "check", True):
+            self.assertFalse(fun(rule_mod))
+
+    @unittest.skipIf(not mock, "unittest.mock is not available")
+    @mock.patch('pkg_resources.iter_entry_points')
```

Comment:
Tests fail on Python 2.7 because of this change I just pushed (because `mock == False` on Python 2).

To solve it, I see at least 2 options:
1. Reintroduce your code in `.travis.yml` (from https://github.com/adrienverge/yamllint/pull/315#discussion_r495436352).
2. Use another way to call mock, for example:
   ```python
   def test_load_plugin_rules_itr(self):
       with mock.patch('pkg_resources.iter_entry_points') as iter_entry_points:
           iter_entry_points.return_value = []
   ```

Reply:
I chosen 2 because Python 2.7 support might be stopped not too far ahead.

Reply:
OK! As you prefer.

Indeed all Python 2 related code will be dropped in 3 months.

Reply:
Thanks, I pushed ca6274e for this.